packages/shared/providers.yaml

```diff
@@ -902,10 +902,11 @@ pipedrive:
         base_url: ${connectionConfig.api_domain}/api
         decompress: true
         paginate:
-            type: offset
-            offset_name_in_request: start
-            response_path: data
+            type: cursor
```

Comment:
It might be dangerous to override an existing pagination system, in case other people are relying on it.

@khaliqgant you added the previous pagination system (based on git blame). Maybe Pipedrive has different pagination per endpoint?

I'm starting to have second thoughts about having the pagination defined at the API level. Maybe we should keep it fully in scripts and consider it's an endpoint-level config.

Reply:
Thank you @bastienbeurier for the insight, I will make the necessary changes and update the PR.